sky/cli.py

```diff
@@ -3700,6 +3700,46 @@ def spot_logs(name: Optional[str], job_id: Optional[int], follow: bool,
         sys.exit(1)
 
 
+@spot.command('dashboard', cls=_DocumentedCodeCommand)
+@usage_lib.entrypoint
```

Comment:
We can have a `--port` option to specify which local port to be used for the port forwarding. If not specified, we do the 5000 and fallback to random free port. (Following the convention like `jupyter-lab`, `tensorboard`, etc)

Reply:
Added.